src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public String getDescription() {
```

Comment:
This method is not used anywhere in the program. Maybe you might want to invoke it whenever you need to access the "description" variable but since you have used the toString() method to achieve that purpose, you can omit this method.